app/models/author.rb

```diff
@@ -6,9 +6,11 @@ class Author < ActiveRecord::Base
   scope :published, -> {
     joins(:zines)
       .merge(Zine.published)
-      .except(:order)
       .distinct('authors.id')
-      .order('name DESC')
+  }
+
+  scope :order_by_name, -> {
+    order('name asc')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.